Desktop/mainwindow.cpp

```diff
@@ -1470,42 +1470,28 @@ void MainWindow::openGitHubBugReport() const
 		}
 	}
 
-	std::stringstream fillIt;
-
-	try			{ fillIt << "* JASP version: " << AppInfo::version.asString()	<< std::endl; }
-	catch(...)	{ fillIt << "* JASP version: ???\n"; }
-
-	try			{ fillIt <<	"* OS name and version: " << QSysInfo::prettyProductName() << std::endl; }
-	catch(...)	{ fillIt << "* OS name and version: ???\n"; }
-
-	fillIt	<<	"<!--- Please fill in the following fields: -->\n"
-				"* Analysis: \n"
-				"* Bug description:\n"
-				"* Expected behaviour:\n"
-				"<!--- Steps to reproduce means, what actions should we take in JASP to reproduce the bug you encountered? --->\n"
-				"#### Steps to reproduce:\n"
-				"1. Go to '...'\n"
-				"2. Click on '....'\n"
-				"3. Scroll down to '....'\n"
-				"4. See error\n";
-
-	fillIt <<	"\n\n\n"
-				"-----------------------------------------------------------------------\n"
-				"<!--- A note from the developers:\nIf possible please attach your data and/or JASP file to the issue, this makes solving the bug a lot easier."
-				" If you would prefer to not make your data publicly available then you could also mail us.\n"
-				"Note that github requires you to zip the file to upload it here.\n-->\n\n"
-				"### Debug information:\n" << _engineSync->currentStateForDebug();
-
-	try			{ fillIt << "\n[Commit used](" << AboutModel::commitUrl() << ")\n"; }
-	catch(...)	{ fillIt << "Commit couldn't be found\n"; }
+	std::stringstream systemInfo, debugInfo;
+
+	try			{ systemInfo << "* JASP version: " << AppInfo::version.asString()	<< std::endl; }
+	catch(...)	{ systemInfo << "* JASP version: ???" << std::endl; }
+
+	try			{ systemInfo <<	"* OS name and version: " << QSysInfo::prettyProductName() << std::endl; }
+	catch(...)	{ systemInfo << "* OS name and version: ???" << std::endl; }
+
+	try			{ systemInfo << "* Commit used: " << AboutModel::commitUrl() << std::endl; }
+	catch(...)	{ systemInfo << "Commit couldn't be found\n"; }
+
+	try			{ debugInfo << "Debug information: " << _engineSync->currentStateForDebug() << std::endl; }
+	catch(...)	{ debugInfo << "No debug information found"; }
 
 	try
 	{
-		QString percentEncodedIssue = QUrl::toPercentEncoding(tq(fillIt.str()));
+		QString systemInfoStr	= QUrl::toPercentEncoding(tq(systemInfo.str())),
+				debugInfoStr	= QUrl::toPercentEncoding(tq(debugInfo.str()));
 
-		const char * baseIssueUrl = "https://github.com/jasp-stats/jasp-issues/issues/new?labels=bug&title=JASP+crashed&body=";
+		QString baseIssueUrl = "https://github.com/jasp-stats/jasp-issues/issues/new?template=crash-report.yml&title=JASP+crashed";
 
-		QUrl issueUrl = baseIssueUrl + percentEncodedIssue;
+		QUrl issueUrl = baseIssueUrl + "&system-info=" + systemInfoStr + "&log=" + debugInfoStr;
```

Comment:
GitHub rest api does not support fill into issue body sub fields, maybe we can just also push .log files and debug info into clipboard and let user paste to here.

Reply:
You can fill it with query parameters. Try this for example (I have generated this by forcing a crash in JASP):
https://github.com/jasp-stats/jasp-issues/issues/new?template=crash-report.yml&title=JASP+crashed&system-info=%2A%20JASP%20version%3A%200.19.3%0A%2A%20OS%20name%20and%20version%3A%20macOS%20Sequoia%20%2815.1%29%0A%2A%20Commit%20used%3A%20https%3A%2F%2Fgithub.com%2Fjasp-stats%2Fjasp-desktop%2Fcommit%2F3183d55fd4daea76858c14b3eefe25c1aeb05d07%0A&log=Debug%20information%3A%20%2A%2AEngine%20%23%200%2A%2A%20process%20%20is%20%2Arunning%2A%20and%20it%27s%20state%20is%20%2Aidle%2A.%0A%0A

Reply:
That's right, but the user still needs to actively find the .log file and upload it, but in fact we have not yet found any user who can effectively find and upload it. My intention is to automatically navigate to the folder where the .log is located (if the user has enabled logging) and push the file to the user's clipboard, or directly open that folder and guide the user to upload the log file. Otherwise we still receive invalid reports, which is no different from whether there is this PR or not😝

Reply:
Yes, that would be better if the log could be (semi) automatically added to the issue. But the first step is to use the right template, and this PR does this.

Reply:
How about this:
1. get path from `AppDirs::logDir`
2. read latest .log files as text stream
3. Push to clipboard or just append to issueUrl **OR** ask in the message box and auto open the `AppDirs::logDir` for user to drag drop to comments.

Reply:
We cannot copy the content of the log files, because the URL would be too long.
But I have tried to help the user to find the log files, by telling which files he should load, and by opening a finder at the logDir path.